support/tech-resources/smtp-rest-api-performance.md

```diff
@@ -9,13 +9,19 @@ SparkPost wants to make sure all of our customers have the best sending experien
 
 We recommend that you send via the Transmission REST API, if possible. Overall the REST API is more efficient than SMTP and requires less resources on the client side.
 
-* Keep the size of the transmission to approximately 50MB.  This refers to the total size of the JSON used for transmission, including any substitution data.
+* On timeout or any 5XX error the best practice is to retry.
+* When retrying, leave at least 5 seconds between retries. We also recommend using exponential backoff.
```

Comment:
@mostfunkyduck had this feedback on Slack, which I agree with and need to incorporate:

> 5 seconds between retries with exponential backoff sounds contradictory, should be 5 seconds for the initial retry, followed by an exponential backoff shouldn't it?